opencti-platform/opencti-front/src/private/components/observations/stix_cyber_observables/StixCyberObservableCreation.jsx

```diff
@@ -227,110 +233,279 @@ const StixCyberObservableCreation = ({
   const { isVocabularyField, fieldToCategory } = useVocabularyCategory();
   const { booleanAttributes, dateAttributes, multipleAttributes, numberAttributes, ignoredAttributes } = useAttributes();
   const [status, setStatus] = useState({ open: false, type: type ?? null });
-
+  const [openProgressDialog, setOpenProgressDialog] = useState(false);
   const handleOpen = () => setStatus({ open: true, type: status.type });
   const localHandleClose = () => setStatus({ open: false, type: type ?? null });
   const selectType = (selected) => setStatus({ open: status.open, type: selected });
+  const [genericValueFieldDisabled, setGenericValueFieldDisabled] = useState(false);
+  const bulkAddMsg = t_i18n('Multiple values entered. Edit with the TT button');
+  const [genericValueFieldValue, setGenericValueFieldValue] = React.useState('');
+  const [bulkValueFieldValue, setBulkValueFieldValue] = React.useState('');
+  const [openBulkModal, setOpenBulkModal] = React.useState(false);
+  let totalObservables = 0;
```

Comment:
I don't see how this variable can be used as it is re-instantiate to 0 at every render

Reply:
Resolved in latest commit. `totalObservables` is now instantiated in the `onSubmit` function which is the only place it is actually used. de7361299907a348c3c013bda4e9fd5c7128b56b